TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -466,7 +466,7 @@ public static void UnqueueCommand(string user, bool isWhisper, [Group(1)] bool u
 	/// <name>Call Command</name>
 	/// <syntax>call (name)\ncallnow (name)</syntax>
 	/// <summary>Calls a command from the queue. callnow skips the requirement set by Call Set. If (name) is specified calls a named command instead of the next command in the queue.</summary>
-	[Command(@"call( *now)?(?! *all| *set| *count)( +.+)?")]
+	[Command(@"call( *now))( +.+)?")]
```

Comment:
```suggestion
	[Command(@"call( *now)?( +.+)?")]
```
You accidentally removed the `?` used for the `( *now)` group and left in a `)`.